hs_core/templates/pages/groups-authenticated.html

```diff
@@ -282,6 +289,37 @@ <h4 class="modal-title" id="myModalLabel">Create New Group</h4>
         </div>
     </div>
 
+    <!-- Explanation Modal -->
```

Comment:
Consider using `include` rather than copying the modal to another template.\

Reply:
done thanks